addons/base_import_module/controllers/main.py

```diff
@@ -15,21 +15,17 @@ def wrap(*args, **kw):
 class ImportModule(Controller):
 
     def check_user(self, uid=None):
-        if uid is None:
-            uid = request.uid
-        is_admin = request.registry['res.users']._is_admin(request.cr, uid, [uid])
-        if not is_admin:
+        if not request.env.user._is_admin():
```

Comment:
why checking with environment's user, why not checking with uid passed in method parameter ? what if I run deploy command with demo user ?
Try with this scenario: run your server, open new terminal and run following command
```./odoo.py deploy --db=your_loaded_db(with base_import_module installed) --login=demo --password=demo path='module directory path'```